### PR TITLE
Convert stroke uniform demo

### PR DESCRIPTION
**Pull request by asturur**

### CODE REVIEW

File: src/content/demo/stroke-uniform-property/code.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+export const code = `
+const canvas = new fabric.Canvas(canvasEl);
+
+fabric.Object.prototype.noScaleCache = false;
```
Comment (ShaMan123) [inline src/content/demo/stroke-uniform-property/code.js:4]: this can't be correct

**Review by ShaMan123 — COMMENTED** [review]
?

File: src/content/demo/stroke-uniform-property/code.js (hunk 1 above)
Comment (ShaMan123) [reply src/content/demo/stroke-uniform-property/code.js:4]: should be via `getDefaults` or `ownDefault`

**Comment by asturur** [comment]
it was super late and i wrote code i forgot to commit, i realized this morning

**Pull request merged**